### DISCUSSION TITLE
是否支持Android Gradle Plugin 7.4.2版本

### DISCUSSION THREAD

**Discussion by oemers** [wurensen/gradle_plugin_android_aspectjx#78] [Q&A]
我用7.4.2版本 编译没问题，运行起来报错

Type com.package.AppManager$1 is defined multiple times: 
/Users/user/Documents/workspace/awesome-android/Demo/app-demo/build/intermediates/mixed_scope_dex_archive/debug/out/0bdd5689f2422654982d4cdcc903fcb88766db2be06620fa1a4849482ecdacca_0.jar:classes.dex, 
/Users/user/Documents/workspace/awesome-android/Demo/app-demo/build/intermediates/mixed_scope_dex_archive/debug/out/4d6f42ee2858d3f0157bb4ffb5a29ad2b569ced76ebe6fcdeb6df291f19779c0_0.jar:classes.dex

**Comment by wurensen** [comment]
@oemers  com.package.AppManager$1 这个类重复了，找找看都出现在什么lib。

**Comment by oemers** [comment]
找到了，是demo多引入了，我运行起来了，我试试能正常hook不

**Comment by oemers** [comment]
您好，我这边上hook的 onCreate方法
我运行起来了， 但hook的内容没有生效， 看了 weaveTmp文件里input什么也没有， buildConfigCache.json里也没有相关的类
但反编译 dex确看到 相关的Activity里有 织入的代码

**Comment by oemers** [comment]
![WX20231012-194312](https://github.com/wurensen/gradle_plugin_android_aspectjx/assets/5078768/0e79156f-07b8-488b-8157-bc7993657700)

**Comment by oemers** [comment]
![WX20231012-194428](https://github.com/wurensen/gradle_plugin_android_aspectjx/assets/5078768/57e38415-029d-4e7f-ba9e-ea32a2c42398)
